Entities/GlassBerry.cs

```diff
@@ -0,0 +1,376 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using Monocle;
+using Celeste.Mod.Entities;
+
+/*
+ * Glass Berry (Spring Collab 2020)
+ * https://github.com/EverestAPI/SpringCollab2020/
+ * 
+ * A custom Strawberry which can be collected normally.
+ * It will break and return to its home location if the player dashes while carrying it.
+ */
+
+namespace Celeste.Mod.SpringCollab2020.Entities {
+    // This custom Strawberry is tracked.
+    // It will appear in the pause menu tracker, as well as all places where red berries are tallied.
+    // This custom Strawberry does not "block collection".
+    // It has normal collection rules, so it can be part of the berry train chain.
+    [RegisterStrawberry(true, false)]
+    [CustomEntity("SpringCollab2020/glassBerry")]
+    class GlassBerry : Entity, IStrawberry, IStrawberrySeeded {
+        // Requested implementations for using IStrawberrySeeded. 
+        // These are the most basic implementations thereof and there is likely no reason to change these.
+        public List<GenericStrawberrySeed> Seeds { get; }
+        public string gotSeedFlag { get { return "collected_seeds_of_" + ID.ToString(); } }
```

Comment:
```suggestion
        public string gotSeedFlag => "collected_seeds_of_" + ID.ToString();
```

Reply:
This isn't something I really learned about yet; I'll make a note of actually learning the thing.